Tensile/KernelWriter.py

```diff
@@ -1090,6 +1169,140 @@ def setupNewTile(self, kernel, tensorParametersA, tensorParametersB, isPap, isOp
 
     return kl
 
+  ##############################################################################
+  # noGlobalLoadLoop
+  # Create the No Global Load Loop (NLL)
+  ##############################################################################
+  def noGlobalLoadLoop( self, kernel, tensorParametersA, tensorParametersB, pack ):
+    kl = []
+    pflr     = self.numItersPLR
+    localWriteEndIter = kernel["LoopIters"] - self.numItersPLR - 1
+
+    kl.append(self.comment3(" NoGlobalLoadLoop - Begin"))
+
+    if not self.numItersPLR:
+      if self.enable["Wait"]:
+        kl.append(self.wait(kernel, tensorParametersA, tensorParametersB, -1, 0, -1, "4wait for local write"))
+      if self.enable["Sync"]:
+        kl.append(self.syncThreads(kernel))
+
+    self.perIterLocalWriteCode = self.perIterLocalWriteCodeNGLL
+    self.perIterLocalWriteCanSkip = [ 0 for i in range (kernel["LoopIters"]) ]
+
+    for u in range(0, kernel["LoopIters"]):
+      kl.append(self.comment("iter %u"%u))
+      plrIdx = (u+pflr) % (self.numVgprBuffer+1) % kernel["LoopIters"]
+      plrPIdx = (u+pflr) % (self.numVgprBuffer+1) % kernel["LoopIters"]
```

Comment:
`plrIdx` is given the exact same value assignment as `plr*P*Idx`. Maybe that is some typo?

Reply:
removed plrPIdx

Reply:
Looks good